test/support/utils_invoke.h

```diff
@@ -219,6 +219,16 @@ struct invoke_on_all_hetero_policies
             auto my_policy = make_new_policy<kernel_name>(queue);
             iterator_invoker<::std::random_access_iterator_tag, /*IsReverse*/ ::std::false_type>()(
                 my_policy, op, ::std::forward<Args>(rest)...);
+
+#if !__SYCL_UNNAMED_LAMBDA__
```

Comment:
With `ONEDPL_USE_UNNAMED_LAMBDA`==`ON` option this check simple doesn't make sense.

Reply:
Could you add a comment describing the purpose of the check? I mean the whole new branch with `iterator_invoker`.
It is very hard to guess it without exploring the PR on GitHub.

Reply:
Done.